nuxt.config.ts

```diff
@@ -0,0 +1,22 @@
+import routes from "./config/routes";
+
+// https://nuxt.com/docs/api/configuration/nuxt-config
+export default defineNuxtConfig({
```

Comment:
Don't forget to add the current [`head` properties](https://nuxt.com/docs/api/configuration/nuxt-config#head) for theming, analytics and performance: 
https://github.com/voorhoede/tudelft-spacefinder/blob/0d281337fecca7e2984695e9957f21460d8921a5/config/nuxt/head.js

Reply:
Yeah. meta name = theme-color is now added by the "new" PWA module. It also adds link rel = icon and doesn't respect it if we add it here manually, so I guess we can live with the icon it provides. Nuxt 3 also defaults on charset and viewport. So only DNS prefetch, site analytics and the default title survived. Please resolve this conversation if it sounds OK :)